packages/graphql-key-transformer/src/__tests__/KeyTransformer.test.ts

```diff
@@ -274,3 +274,80 @@ function getInputType(doc: DocumentNode, type: string): InputObjectTypeDefinitio
     | InputObjectTypeDefinitionNode
     | undefined;
 }
+
+describe('check schema input', () => {
+  let ff: FeatureFlagProvider;
+  beforeEach(() => {
+    ff = {
+      getBoolean: jest.fn().mockImplementation((name, defaultValue) => {
+        if (name === 'skipOverrideMutationInputTypes') {
+          return true;
+        }
+      }),
+      getNumber: jest.fn(),
+      getObject: jest.fn(),
+      getString: jest.fn(),
+    };
+  });
+
+  it('@model mutation with user defined null args ', () => {
+    const validSchema = /* GraphQL */ `
+      type Call
+        @model(queries: null, mutations: null)
+        @key(fields: ["receiverId", "senderId"])
+        @key(name: "bySender", fields: ["senderId", "receiverId"]) {
+        senderId: ID!
+        receiverId: ID!
+      }
+
+      type Mutation {
+        createCall(input: CreateCallInput!): Call
+        deleteCall(input: DeleteCallInput!): Call
+      }
+
+      input CreateCallInput {
+        receiverId: ID!
+      }
+
+      input DeleteCallInput {
+        receiverId: ID!
+      }
+    `;
+    const transformer = new GraphQLTransform({
+      transformers: [new DynamoDBModelTransformer(), new KeyTransformer()],
+      featureFlags: ff,
+    });
+    const out = transformer.transform(validSchema);
+    expect(out).toBeDefined();
+    const schema = out.schema;
+    expect(schema).toMatchSnapshot();
+  });
+
+  it('@model mutation with user defined create args ', () => {
+    const validSchema = /* GraphQL */ `
+      type Call
+        @model(queries: null, mutations: { delete: "testDelete" })
+        @key(fields: ["receiverId", "senderId"])
+        @key(name: "bySender", fields: ["senderId", "receiverId"]) {
+        senderId: ID!
+        receiverId: ID!
+      }
+
+      input CreateCallInput {
+        receiverId: ID!
+      }
+
+      input DeleteCallInput {
+        receiverId: ID!
+      }
+    `;
+    const transformer = new GraphQLTransform({
+      transformers: [new DynamoDBModelTransformer(), new KeyTransformer()],
+      featureFlags: ff,
+    });
+    const out = transformer.transform(validSchema);
+    expect(out).toBeDefined();
+    const schema = out.schema;
+    expect(schema).toMatchSnapshot();
```

Comment:
Can you add proper assertion along with snapshot tests. With proper assertion it makes 100% which part of the generated schema we are testing and prevents accidental update to the snapshots

Reply:
replace with a check on input type